my-app/src/Components/PersonForm.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react';
+import { withFormik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import axios from 'axios';
+
+const PersonForm = ({ values, errors, touched, status }) => {
+  console.log(values);
+  console.log(errors);
+  console.log(touched);
+
+  const [person, setPerson] = useState([]);
+
+  // added useEffect in order to see the changes happening after we submit the form
+  useEffect(() => {
+    // console.log(`status has been updated`, status);
+    // rendering the data we got after the form was submitted
+    status && setPerson(person => [...person, status]);
+  }, [status])
+  return (
+    <div className='style'>React Formik
+      <Form className='column'>
+        <Field id='name' type='name' name='username' placeholder='Username' />
+        {touched.username && errors.username && (                       //Validation syntax from YUP
+          <p>*{errors.username}</p>
+        )}
+        <Field id='password' type='password' name='password' placeholder='Password' />
+        {touched.password && errors.password && (                       //Validation syntax from YUP
+          <p>*{errors.password}</p>
+        )}
+        <Field id='email' type='email' name='email' placeholder='Email' />
+        {touched.email && errors.email && (                             //Validation syntax from YUP
+          <p>*{errors.email}</p>
+        )}
+
+        <Field as='select' id='role' name='role'>
+          <option disabled>Choose a role</option>
+          <option value='UI-Dev'>UI Dev</option>
+          <option value='UX-Dev'>UX Dev</option>
+          <option value='ReactI-Dev'>React I Dev</option>
+          <option value='ReactII-Dev'>React II Dev</option>
+          <option value='Backend-Dev'>Backend Dev</option>
+        </Field>
+
+        <label> Terms and conditions
+          <Field type='checkbox' name='terms' checked={values.terms} />
+        </label>
+        <button type='submit'>Submit!</button>
+      </Form>
+
+      {/* adding structure in order to save the data users are putting in */}
+      {person.map(e => {
```

Comment:
Variable name should have meaningful value (e is not). Should be:
persons.map(person => {})